persistence/play-jdbc-evolutions/src/main/scala/play/api/db/evolutions/Evolutions.scala

```diff
@@ -224,15 +239,30 @@ object Evolutions {
    * @param autocommit Whether to use autocommit or not, evolutions will be manually committed if false.
    * @param schema The schema where all the play evolution tables are saved in
    * @param metaTable Table to keep evolutions' meta data
+   * @param substitutionsMappings Mappings of variables (without the prefix and curly braces) and their replacements.
+   * @param substitutionsPrefix Prefix of the variable to substitute. Will be combined with curly braces, e.g. "${my_variable}".
+   * @param substitutionsEscape Whetever escaping of variables is enabled via the syntax "${!...}".
+   *     E.g. "${!my_variable}" ends up as "${my_variable}" in the final sql instead of replacing it
+   *     with it's substitution.
```

Comment:
```suggestion
   *     with its substitution.
```

The possessive "its" is used when referring to an ownership, "it's" is a contraction of "it is".